js/datetime/Index.svelte

```diff
@@ -108,6 +113,7 @@
 				bind:this={datetime}
```

Comment:
Hi @yinsumirage - what about setting `disabled={!interactive}` in the `input` components as opposed to making the click events no-ops? I think that's more consistent with our other components, like textbox